src/install-webview.js

```diff
@@ -4,8 +4,7 @@ const os = require('os');
 const vscode = require('vscode');
 const {workspace} = vscode;
 const formidable = require('formidable');
```

Comment:
unused?

Reply:
nice catch, indeed we no longer need that as it was used to parse form data sent by the page when we had to rely on a server started by the extension.